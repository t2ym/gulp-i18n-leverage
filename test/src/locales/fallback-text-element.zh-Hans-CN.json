{
  "model": {},
  "text": "zh-Hans-CN  outermost text at the beginning ",
  "h1_3": "zh-Hans-CN outermost header 1",
  "text_4": "zh-Hans-CN  outermost text in the middle ",
  "span_5": "zh-Hans-CN simple text without id",
  "span_6": "zh-Hans-CN simple text without id 2",
  "label-1": "zh-Hans-CN simple text with id",
  "label-2": "zh-Hans-CN simple text with id 2",
  "div_10:span_1": "zh-Hans-CN simple text within div",
  "toplevel-div:span": "zh-Hans-CN simple text within div",
  "toplevel-div:span_1": "zh-Hans-CN simple text within div 2",
  "third-level-div": "zh-Hans-CN great grandchild text within div",
  "second-level-div:div_1": "zh-Hans-CN great grandchild text within div without id",
  "p_13": [
    "zh-Hans-CN A paragraph with {1} is converted to {2}.",
    "zh-Hans-CN parameters",
    "zh-Hans-CN <i18n-format>"
  ],
  "paragraph": [
    "zh-Hans-CN A paragraph with {1} is converted to {2}.",
    "zh-Hans-CN id",
    "zh-Hans-CN <i18n-format>"
  ],
  "text_15": "zh-Hans-CN  outermost text at the end "
}
