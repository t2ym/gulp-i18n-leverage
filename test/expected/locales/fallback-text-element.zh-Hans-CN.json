{
  "meta": {
    "todo": [
      {
        "op": "add",
        "path": "/div_9:span",
        "value": "simple text within div"
      },
      {
        "op": "add",
        "path": "/div_9:span_1",
        "value": "simple text within div 2"
      },
      {
        "op": "add",
        "path": "/div_9:div_2:div",
        "value": "great grandchild text within div"
      },
      {
        "op": "add",
        "path": "/div_10:text",
        "value": " simple text as the first element in div "
      },
      {
        "op": "add",
        "path": "/div_10:text_2",
        "value": " simple text in the middle of div "
      },
      {
        "op": "add",
        "path": "/div_10:span_3",
        "value": "simple text within div 2"
      },
      {
        "op": "add",
        "path": "/div_10:div_4:div",
        "value": "great grandchild text within div"
      },
      {
        "op": "add",
        "path": "/div_10:text_5",
        "value": " simple text at the last element in div "
      },
      {
        "op": "add",
        "path": "/div_12:ul:li",
        "value": "line item without id 1"
      },
      {
        "op": "add",
        "path": "/div_12:ul:li_1",
        "value": "line item without id 2"
      },
      {
        "op": "add",
        "path": "/div_12:ul:li_2",
        "value": "line item without id 3"
      },
      {
        "op": "add",
        "path": "/line-items:li",
        "value": "line item with id 1"
      },
      {
        "op": "add",
        "path": "/line-items:li_1",
        "value": "line item with id 2"
      },
      {
        "op": "add",
        "path": "/line-items:li_2",
        "value": "line item with id 3"
      }
    ]
  },
  "model": {},
  "text": "zh-Hans-CN  outermost text at the beginning ",
  "h1_3": "zh-Hans-CN outermost header 1",
  "text_4": "zh-Hans-CN  outermost text in the middle ",
  "span_5": "zh-Hans-CN simple text without id",
  "span_6": "zh-Hans-CN simple text without id 2",
  "label-1": "zh-Hans-CN simple text with id",
  "label-2": "zh-Hans-CN simple text with id 2",
  "div_9:span": "simple text within div",
  "div_9:span_1": "simple text within div 2",
  "div_9:div_2:div": "great grandchild text within div",
  "div_10:text": " simple text as the first element in div ",
  "div_10:span_1": "zh-Hans-CN simple text within div",
  "div_10:text_2": " simple text in the middle of div ",
  "div_10:span_3": "simple text within div 2",
  "div_10:div_4:div": "great grandchild text within div",
  "div_10:text_5": " simple text at the last element in div ",
  "toplevel-div:span": "zh-Hans-CN simple text within div",
  "toplevel-div:span_1": "zh-Hans-CN simple text within div 2",
  "third-level-div": "zh-Hans-CN great grandchild text within div",
  "second-level-div:div_1": "zh-Hans-CN great grandchild text within div without id",
  "div_12:ul:li": "line item without id 1",
  "div_12:ul:li_1": "line item without id 2",
  "div_12:ul:li_2": "line item without id 3",
  "line-items:li": "line item with id 1",
  "line-items:li_1": "line item with id 2",
  "line-items:li_2": "line item with id 3",
  "p_13": [
    "zh-Hans-CN A paragraph with {1} is converted to {2}.",
    "zh-Hans-CN parameters",
    "zh-Hans-CN <i18n-format>"
  ],
  "paragraph": [
    "zh-Hans-CN A paragraph with {1} is converted to {2}.",
    "zh-Hans-CN id",
    "zh-Hans-CN <i18n-format>"
  ],
  "text_15": "zh-Hans-CN  outermost text at the end "
}